CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- added in colour variables for discovery and live badges and banners
+- updated link colour variables to match gov.uk [#795](https://github.com/hmrc/assets-frontend/pull/795)
```

Comment:
Are these the same thing? Can you consolidate them if they are, or make sure they both have PR links if not.

Also, can you make sure to capitalise for consistency please?